README.md

```diff
@@ -27,16 +35,24 @@ Curiosity Chrome extension is available now. Which is made by [Aswanth](https://
 
 I've collected usernames of amazing stargazers :star: in Github. By using Github API, Curiosity will collect their stars and will show as a feed.
 
-## What you need
-
-You need to submit Github Token to access Github API.
 
-## Tools I used
+## Dependencies (libraries/frameworks/tools)
 
 * [SweetAlert2](https://limonte.github.io/sweetalert2/)
 * [infinite-scroll](https://github.com/alexblack/infinite-scroll)
 * [Axios](https://github.com/mzabriskie/axios)
 
+## Labels
+* `up-for-grabs` labels are the issues need to solved. 
+* `discussion` label is for discussions of tasks. 
+* `enhancement` are features to make project better.
+* `testing` are features that aren't implemented yet but are being tested.
```

Comment:
Testing is not for the future features. It's for unit testing, code testing and code coverage.